Automattic/vip-go-mu-plugins
# Update lightweight term count

Brings in 2 changes.
#15 which changes from passing the taxonomy name to the taxonomy object as expected by the callback functions and

#14 which fixes a bug where all terms would be processed if a term had already been processed. Slowing down the site dramatically

## Checklist


## Steps to Test

Add or remove term to post and check on term page to ensure the term count is accurate.
